checker/src/main/java/org/checkerframework/checker/interning/InterningAnnotatedTypeFactory.java

```diff
@@ -65,6 +72,62 @@ public InterningAnnotatedTypeFactory(BaseTypeChecker checker) {
         this.postInit();
     }
 
+    @Override
+    protected DefaultQualifierForUseTypeAnnotator createDefaultForUseTypeAnnotator() {
+        return new InterningDefaultQualifierForUseTypeAnnotator(this);
+    }
+
+    /**
+     * Does not add defaults for type uses on constructor results. Constructor results should be
+     * {@code @UnknownInterned} by default.
+     */
+    static class InterningDefaultQualifierForUseTypeAnnotator
+            extends DefaultQualifierForUseTypeAnnotator {
+
+        public InterningDefaultQualifierForUseTypeAnnotator(AnnotatedTypeFactory typeFactory) {
+            super(typeFactory);
+        }
+
+        @Override
+        public Void visitExecutable(AnnotatedExecutableType type, Void p) {
+            MethodSymbol methodElt = (MethodSymbol) type.getElement();
+
+            if (methodElt == null
+                    || methodElt.getKind() != ElementKind.CONSTRUCTOR
+                    || methodElt.owner.getKind() == ElementKind.ENUM) {
+                // Annotate method returns, not constructors.
```

Comment:
`... not constructors or enums`? There is a check for enums, too.

Reply:
Enum check was wrong.  Thanks!